sdk/table/azure-table/README.md

```diff
@@ -189,7 +189,7 @@ from azure.table.aio import TableClient
 
 my_entity = {'PartitionKey':'part','RowKey':'row'}
```

Comment:
we don't have to do this now, but might be interesting to have some other keys as part of the entity:
```
    entity = {
        'PartitionKey': 'color',
        'RowKey': 'brand',
        'text': 'Marker',
        'color': 'Purple',
        'price': '5'
    }
```